web/build.mjs

```diff
@@ -10,11 +10,15 @@ import { fileURLToPath } from "url";
 
 const __dirname = fileURLToPath(new URL(".", import.meta.url));
 
-// Use the package.json file in the root folder, as it has the current version information.
-const authentikProjectRoot = execFileSync("git", ["rev-parse", "--show-toplevel"], { encoding: "utf8" }).replace(
-    "\n",
-    ""
-);
+let authentikProjectRoot = __dirname + "../";
+try {
+    // Use the package.json file in the root folder, as it has the current version information.
+    authentikProjectRoot = execFileSync("git", ["rev-parse", "--show-toplevel"], {
+        encoding: "utf8",
+    }).replace("\n", "");
+} catch (exc) {
+    // We probably don't have a .git folder, which could happen in container builds
+}
```

Comment:
Oh! I wasn't aware there wasn't a clearly defined project root in container builds. In that case you're right and we shouldn't even bother with using the `git rev-parse` trick to find it; I would just remove the whole thing and live with what's on line 13.